src/explore-education-statistics-frontend/src/modules/table-tool/components/TableToolShare.tsx

```diff
@@ -16,13 +19,14 @@ const linkInstructions =
 
 interface Props {
   tableHeaders?: TableHeadersConfig;
-  query: TableDataQuery;
+  query: ReleaseTableDataQuery;
 }
 
 const TableToolShare = ({ tableHeaders, query }: Props) => {
   const [permalinkUrl, setPermalinkUrl] = useState('');
   const [permalinkLoading, setPermalinkLoading] = useState<boolean>(false);
   const [screenReaderMessage, setScreenReaderMessage] = useState('');
+  const [permalinkError, togglePermalinkError] = useToggle(false);
```

Comment:
Would put this next to the other permalink states for better organisation

Reply:
Done in combination with change to use normal state rather than toggle.